load-generator/README.md

```diff
@@ -15,8 +15,16 @@
     1. --dataFormat -d (eg, otlp, prometheus, xray)
 
 ### OTLP Metrics Load Test Sample Command,
+
+The following is a list of optional command line flags for configuration of OTLP metrics:
+* `--flushInterval, -f` : (default=1000)the metric collection export interval
+* `--metric_count, -m`: (default=1) the number of metrics that should be created for each metric type
+* `--datapointCount, -dp`: (default=1) the number of datapoints that should be created for each metric.
```

Comment:
Are these per-period?  How do these values relate to the other values?

Reply:
Thanks. Updated readme with this information.